R/simmicmenten.R

```diff
@@ -0,0 +1,42 @@
+simmicmenten <- function(x, vmax, km, cv, reps, h = 1, log=F) {
+
+  y = vmax*x^h/(km^h + x)
+
+  values <- data.frame(x=rep(x, reps), y)
+
+  values <- dplyr::mutate(values, y=apply(values, 1, function(x) stats::rnorm(1, x[2], cv*x[2])))
+
+  for (i in 1) {
+    if (log) {
+      model <- stats::lm(log(y) ~ log(x), data=values)
```

Comment:
No. there is no need to log y values. Only the x value gets logged.

Reply:
These log values are just for the nls() call right after. Log y does not get plotted. I described this in more detail on the sim1sbind pull request thread.